README.md

```diff
@@ -37,13 +37,32 @@ OpenSearch Dashboards Reports integration with [Notifications](https://github.co
 
 ## Troubleshooting
 
-#### Fail to launch Chromium
+### Fail to launch Chromium
 
 There could be two reasons for this problem
 
 1. You are not having the correct version of headless-chrome matching to the OS that your OpenSearch Dashboards is running. Different versions of headless-chrome can be found [here](https://github.com/opensearch-project/dashboards-reports/releases/tag/chromium-1.12.0.0)
 
 2. Missing additional dependencies. Please refer to [additional dependencies section](./dashboards-reports/rendering-engine/headless-chrome/README.md#additional-libaries) to install required dependencies according to your operating system.
+
+### Missing Font Dependencies
+Chromium may not have all of the dependencies you may require to be able to view all of the content of your reports.
+
+If you are using a CentOS/RHEL system, install the following packages:
+* [`ipa-gothic-fonts`](https://centos.pkgs.org/8/raven-x86_64/ipa-gothic-fonts-003.03-15.el8.noarch.rpm.html)
```

Comment:
why is this package for centos 8 while others are for 7? i don't think there's many people using 8 as its EOL is in less than 2 months

Reply:
I took the packages that were suggested from the corresponding page on the Elastic docs, wasn't aware that centos 8 is being deprecated so soon. Looks like CentOS 7 EOL is not until 2024, can change it to that for now

Reply:
Looks like this package doesn't exist for 7, will need to find an alternate

Reply:
https://centos.pkgs.org/7/centos-x86_64/ipa-gothic-fonts-003.03-5.el7.noarch.rpm.html